explorer/src/components/UniversalTable.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import { printableCoin } from '@nymproject/nym-validator-client';
+import { cellStyles } from './Universal-DataGrid';
+
+export type ColumnsType = {
+  field: string;
+  title: string;
+  headerAlign: string;
+  flex?: number;
+  width?: number;
+};
+
+export interface UniversalTableProps {
+  tableName: string;
+  columnsData: ColumnsType[];
+  rows: any[];
+}
+
+function formatCellValues(val: string | number, field: string) {
+  if (field === 'bond') {
+    return printableCoin({ amount: val.toString(), denom: 'unpunk' });
+  }
+  return val;
+}
+
+export const UniversalTable = ({
+  tableName,
+  columnsData,
+  rows,
+}: UniversalTableProps) => (
+  <TableContainer component={Paper}>
+    <Table sx={{ minWidth: 650 }} aria-label={tableName}>
+      <TableHead>
+        <TableRow>
+          {columnsData?.map(({ field, title, flex }) => (
+            <TableCell key={field} sx={{ fontWeight: 'bold', flex }}>
+              {title}
+            </TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {rows.map((eachRow) => (
+          <TableRow
+            key={eachRow.owner}
```

Comment:
This stops the component from being reuseable as it expects the property `owner` to exists. I suggest either building it as a specialised component and call it what it is being used for `DetailTable` or something like that and then speficy the row prop type OR the other option is to make it reuseable by allowing any row properties to be passed to it and using a generic type.

Reply:
I've changed it to `<DetailTable />` for now along with the <angleBracket> args for TS.